homeassistant/components/pinecil/strings.json

```
@@ -0,0 +1,83 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "[%key:component::bluetooth::config::step::user::description%]",
+        "data": {
+          "address": "[%key:common::config_flow::data::device%]"
+        }
+      },
+      "bluetooth_confirm": {
+        "description": "[%key:component::bluetooth::config::step::bluetooth_confirm::description%]"
+      }
+    },
+    "abort": {
+      "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "live_temperature": {
+        "name": "Tip temperature"
+      },
+      "voltage": {
+        "name": "DC Input Voltage"
+      },
+      "handle_temperature": {
+        "name": "Handle Temperature"
+      },
+      "power_pwm_level": {
+        "name": "Power Level"
+      },
+      "power_source": {
+        "name": "Power Source",
+        "state": {
+          "dc": "DC Input",
+          "qc": "USB Quick Charge",
+          "pd_vbus": "USB PD VBUS",
+          "pd": "USB Power Delivery"
+        }
+      },
+      "tip_resistance": {
+        "name": "Tip Resistance"
```

```suggestion
        "name": "Tip resistance"
```
Please use sentence case